Comments/Controllers/Api/OrganisationController.cs

```diff
@@ -0,0 +1,136 @@
+using Comments.Common;
+using Comments.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.FeatureManagement.Mvc;
+using System;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Comments.Controllers.Api
+{
+	[Produces("application/json")]
+    [Route("consultations/api/[controller]")]
+	[FeatureGate(Constants.Features.OrganisationalCommenting)]
+	public class OrganisationController : ControllerBase
+    {
+	    private readonly IOrganisationService _organisationService;
+        private readonly ILogger<OrganisationController> _logger;
+
+        public OrganisationController(IOrganisationService organisationService, ILogger<OrganisationController> logger)
+        {
+
+	        _organisationService = organisationService;
+	        _logger = logger;
```

Comment:
Do we need to do any logging in this class?

Reply:
i don't think so. i've removed the property.